testcases/PowerNVDump.py

```diff
@@ -1069,6 +1069,55 @@ def runTest(self):
             else:
                 log.info("PASSED: Hugepage size is {} kB".format(hugepage_size))
 
+class KernelCrash_KdumpPMEM(PowerNVDump):
+    '''
+    This test verifies kdump/fadump on pmem device.
+    '''
+
+    def runTest(self):
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.setup_test()
+        if self.distro == "rhel":
+            cmd = "yum -y install ndctl"
+        if self.distro == "sles":
+            cmd = "zypper install -y ndctl"
+        self.c.run_command("%s; sync" % cmd, timeout=120)
+        if not self.c.run_command("ndctl list -Ru"):
+            self.skipTest("Please configure pmem from HMC and try again.")
+        try:
+            self.c.run_command("ndctl create-namespace -m fsdax -r region0")
+        except CommandFailed:
+            try: self.c.run_command("ndctl destroy-namespace all -f")
+            except CommandFailed: 
+                self.c.run_command("umount /pmem0")
+                self.c.run_command("ndctl destroy-namespace all -f")
+                self.c.run_command("ndctl create-namespace -m fsdax -r region0")
+        try: self.c.run_command("mkfs.xfs -f -b size=64k -s size=4k -m reflink=0 /dev/pmem0")
+        except CommandFailed: 
+            self.c.run_command("umount /pmem0")
+            self.c.run_command("mkfs.xfs -f -b size=64k -s size=4k -m reflink=0 /dev/pmem0")
+        self.c.run_command("mkdir -p /pmem0")
+        self.c.run_command("mount -o dax /dev/pmem0 /pmem0")
+        self.c.run_command("echo '/dev/pmem0 /pmem0                   xfs     defaults        0 0' >> /etc/fstab")
+        if self.distro == "rhel":
+            self.c.run_command("echo 'add_drivers+=\"papr_scm\"' > /etc/dracut.conf.d/99-pmem-workaround.conf")
```

Comment:
what is this workaruound ?

Reply:
pmem driver was not added in dump initrd  so i required